test/index.test.js

```diff
@@ -258,4 +262,71 @@ if (isWin32) {
       done();
     });
   });
+
+  describe('windows path with drive or UNC volume', function() {
+    it('should return parent dirname from absolute path with drive letter', function(done) {
+      expect(gp('C:/')).toEqual('C:/');
+      expect(gp('C:/.')).toEqual('C:/');
+      expect(gp('C:/*')).toEqual('C:/');
+      expect(gp('C:/./*')).toEqual('C:/.');
+      expect(gp('C://')).toEqual('C:/');
+      expect(gp('C://*')).toEqual('C:/');
+      expect(gp('C:/path/*.js')).toEqual('C:/path');
+
+      expect(gp('C:\\')).toEqual('C:/');
+      expect(gp('C:\\.')).toEqual('C:/');
+      expect(gp('C:\\*')).toEqual('C:/');
+      expect(gp('C:\\.\\*')).toEqual('C:/.');
+      expect(gp('C:\\\\')).toEqual('C:/');
+      expect(gp('C:\\\\*')).toEqual('C:/');
+      expect(gp('C:\\path\\*.js')).toEqual('C:/path');
+
+      done();
+    });
+
+    it('should return parent dirname from relative path with drive letter', function(done) {
+      expect(gp('C:')).toEqual('C:.');
+      expect(gp('C:.')).toEqual('C:.');
+      expect(gp('C:*')).toEqual('C:.');
+      expect(gp('C:./*')).toEqual('C:.');
+      expect(gp('C:.//')).toEqual('C:./');
+      expect(gp('C:.//*')).toEqual('C:./');
```

Comment:
Why does this add `.` here? Shouldn't it add `/` because `C:` is the root?

Or is this because `:` should be an allowed character in a relative path?